### PR TITLE
feat!: implement missing requests

### PR DESCRIPTION
**Pull request by stanislav-zeman**
This PR adds missing requests to the client that are documented in the Recombee documentation.

### CODE REVIEW

File: item_values.go [hunk 1]
```diff
@@ -22,3 +21,10 @@ func GetItemValues(itemId string) Request {
 		Method: http.MethodGet,
 	}
 }
+
+func UpdateMoreItems() Request {
```
Comment (Matovidlo) [inline item_values.go:26]: Missing godoc.

File: items.go [hunk 2]
```diff
@@ -40,3 +40,10 @@ func ListItems(opts ...RequestOption) Request {
 		Params: params,
 	}
 }
+
+func DeleteMoreItems() Request {
```
Comment (Matovidlo) [inline items.go:49]: same

File: recommandation.go [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package recombee
+
+type Recommendations struct {
```
Comment (Matovidlo) [inline recommandation.go:3]: missing godoc. What is the purpose for this new strucutre. How should be used. Maybe add example test? If not create issue in this project so in godoc is an example of usage of `Recommendations`

File: search.go [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+	}
+
+	return Request{
+		Path:   fmt.Sprintf("/search/users/%s/items/", userId),
+		Method: http.MethodGet,
+		Params: params,
+	}
+}
+
+func SearchItemSegments(userId string, searchQuery string, count int, opts ...RequestOption) Request {
```
Comment (Matovidlo) [inline search.go:26]: missing godoc.

**Review by Matovidlo — CHANGES_REQUESTED** [review]

File: search.go (hunk 4 above)
Comment (stanislav-zeman) [reply search.go:26]: Fixed

File: items.go (hunk 2 above)
Comment (stanislav-zeman) [reply items.go:49]: Fixed

File: item_values.go (hunk 1 above)
Comment (stanislav-zeman) [reply item_values.go:26]: Fixed

File: recommandation.go (hunk 3 above)
Comment (stanislav-zeman) [reply recommandation.go:3]: Issue created: https://github.com/moderntv/recombee/issues/7#issue-2009718738

**Review by Matovidlo — APPROVED** [review]

**Pull request merged**